content/next/index.md

```diff
@@ -88,6 +88,8 @@ per-line.
 1. A description MUST be provided after the `BREAKING CHANGE: `, describing what has changed about the API, e.g., _BREAKING CHANGE: environment variables now take precedence over config files._
 1. Types other than `feat` and `fix` MAY be used in your commit messages.
 1. The units of information that make up conventional commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.
+1. A `!` MAY be appended prior to the `:` in the type/scope prefix, to further draw attention to breaking changes. `BREAKING CHANGE: description` MUST also be included in the body
```

Comment:
We may add an example too for this ☝️

Reply:
`along with the` sounds confusing to me. It may imply lexical adjacency. How about something like:

`or footer, when using the ! prefix.`

Reply:
I think `or footer when using the ! prefix.` (without a comma) sounds good.  I also think just ending the sentence at `or footer.` is good too.

Reply:
**Whatever the in-the-title signifier is for a breaking change, I'd strongly prefer that Conventional Commits specify it as a MUST** rather than a MAY. I'd like to be able to reliably notice breaking changes when scanning only the first lines of commit messages. MAY makes it unreliable.

If it's a MAY: Skimming some changes, I see no signifier in the commit message titles. Oh good, no breaking changes. Except I don't know that. Istill need to look at the full body and footer of all the messages in order to reliably notice all the breaking changes documented by commit messages compliant with Conventional Commits.

If it's a MUST: Skimming some changes, piping their titles through grep even, I see no signifier in the commit message titles. Oh good, no breaking changes.

Reply:
@apetro this is a `MAY` it's because we don't want to introduce a BC across all the tools for something that still in a "test" phase.

The MUST has always been the `BREAKING CHANGE:` in the message body.
If the `!` will be adopted by most of the community then we may think about doing it mandatory.

Reply:
Conventional Commits is currently `v1.0.0-beta.3`. Beta is the opportunity to include the features that the spec needs to be effective. Yes, MUST-ing the sigil is a breaking change, but the time to make a breaking change to this spec is before 1.0.0.

I feel quite strongly that it should be a MUST and that MUST makes the feature a whole lot more valuable than does MAY.

That said, if I can't get you to MUST, maybe it could at least be a SHOULD ?

Reply:
@bcoe ☝️

Reply:
@apetro I'm less convinced on this convention being a necessity (although I bet there's a chance my opinion shifts in the future); I'm hesitant to enforce additional ritual in the spec at this time.

however, I would (personally) be happy to opt for the language `SHOULD` rather than `MUST`. How do other folks feel (@hutson, @damianopetrungaro, @zeke, @ofrobots).

Reply:
I preferiti SHOULD as already explained.
I do not see the necessary of a BC at this stage of the addition.

If in the future tools and ppl will use it then use MUST will be there :)

Reply:
`SHOULD` sounds like a good compromise to me.